frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -23,6 +23,29 @@ export const TargetAreaToName = {
 export type supportTicketTargetArea = keyof typeof TargetAreaToName
 export type supportTicketKind = 'bug' | 'feedback'
 
+export const URLPathToTargetArea: Record<string, supportTicketTargetArea> = {
+    insights: 'analytics',
+    recordings: 'session_reply',
+    dashboard: 'analytics',
+    feature_flags: 'feature_flags',
+    experiments: 'experiments',
+    'web-performance': 'session_reply',
+    events: 'analytics',
+    'data-management': 'data_management',
+    cohorts: 'cohorts',
+    annotations: 'analytics',
+    persons: 'data_integrity',
+    groups: 'data_integrity',
+    app: 'apps',
+    apps: 'apps', // TODO: it currently is project/apps for some reason :shrug
+    toolbar: 'analytics',
+}
+
+export function getURLPathToTargetArea(pathname: string): supportTicketTargetArea | undefined {
+    const first_part = pathname.split('/')[1]
+    return URLPathToTargetArea[first_part]
+}
```

Comment:
To fix the type errors this will need to be:

```suggestion
export function getURLPathToTargetArea(pathname: string): supportTicketTargetArea | null {
    const first_part = pathname.split('/')[1]
    return URLPathToTargetArea[first_part] ?? null
}
```